tests/integration/gcp.py

```diff
@@ -90,6 +86,18 @@ def validate_config(cls, cfg: dict, image: str, test_name: str, credentials):
             cfg['ssh']['ssh_key_name'] = f"key-{test_name}"
         if not 'user' in cfg['ssh']:
             cfg['ssh']['user'] = "gardenlinux"
+        if not 'uefi' in cfg:
+            cfg['uefi'] = False
+        if not 'secureboot' in cfg:
+            cfg['secureboot'] = False
+        if not 'db_path' in cfg:
+            cfg['db_path'] = "/gardenlinux/cert/secureboot.db.auth"
```

Comment:
We should probably group these parameters (so `db_path`, `kek_path`, `pk_path` and `cert_file_type`) underneath the `secureboot` parameter because these parameters only make sense if you also enable `secureboot`. 

We would streamline this with the `ssh` parameter, which does do the same with its parameters.